.github/workflows/python-client.yml

```diff
@@ -83,8 +83,8 @@ jobs:
           mkdir -p .kube/external
           go run github.com/magefile/mage@v1.14.0 -v localdev minimal
       - name: Install Protoc
-        uses: arduino/setup-protoc@v1
+        uses: arduino/setup-protoc@v2
         with:
-          version: '3.17.3'
+          version: '21.0'
```

Comment:
Is this current? I believe we're using protoc 23.3.  Version 24.0 is the very latest, but it only just came out a couple days ago, so we may not want that.

Reply:
Changed it to 23.3